app/index.js

```diff
@@ -97,6 +98,13 @@ if (isDev) {
 } else {
   //eslint-disable-next-line no-console
   console.log('running in prod mode');
+  if (process.platform === 'win32') {
+    //eslint-disable-next-line no-console
+    addBinToUserPath().catch(err => console.error('Failed to add Hyper CLI path to user PATH', err));
```

Comment:
Should this be shown to the user (notification) or not?

Reply:
Not enough place to print error in notification.
On macOS, error could be read in Console.app. But I don't know for other platform

Reply:
Let's keep it as it is and see if it's needed later on 👍  Let's not over engineer it